app/models/host/managed.rb

```diff
@@ -912,22 +912,6 @@ def recreate_config(only = nil)
     result
   end
 
-  # converts a name into ip address using DNS.
-  # if we are managing DNS, we can query the correct DNS server
-  # otherwise, use normal systems dns settings to resolv
-  def to_ip_address(name_or_ip)
```

Comment:
Should we deprecate this first in case some plugin uses it? I didn't find any that does but there may one.